### PR TITLE
Update Read Me

### PR DESCRIPTION
**Pull request by audreylemeur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,95 @@
...
+* Mockito is used to mock various classes to assist in testing.
+* Gradle is used to build the project.
+
+### Database
+* MongoDB is used as our database for storing notes and owners.
+
+### Deployment
+* Docker is used to contain software in its own packages and still allow it to communicate with each other.
+* DigitalOcean is used as a hosting service.
+
```
Comment (ErikRauer) [inline README.md:69]: We should add a section about how Auth0 is used to securely handle the authentication of users.

File: README.md [hunk 2]
```diff
@@ -4,28 +4,95 @@
...
-Features provided by this new code base are as follows:
-* Pinning notes
-* GCAL redirect
-* QR Code
-* Time stamps
 
 ## API
 
- 🐚 The server's API is a bit weird in places. A full description of
+ The server's API is a bit weird in places. A full description of
```
Comment (ErikRauer) [inline README.md:88]: Probably want to remove the sentence about the API being weird in places.

File: README.md [hunk 3]
```diff
@@ -4,28 +4,95 @@
...
  the API is can be found in [HTTP_API.md](HTTP_API.md).
 
  ## Deployment
 
  As always, instructions on how to crate a DigitalOcean Droplet and setup the project are in [DEPLOYMENT.md](DEPLOYMENT.md).
```
Comment (ErikRauer) [inline README.md:93]: Remove "as always" and "setup the project" from the sentence

File: README.md [hunk 4]
```diff
@@ -4,28 +4,95 @@
...
 
- 🐚 If you want to deploy this project on a new droplet, you will need to
+ If you want to deploy this project on a new droplet, you will need to
```
Comment (ErikRauer) [inline README.md:95]: We should add this part to DEPLOYMENT.md rather than have it in the README

File: README.md [hunk 5]
```diff
@@ -4,28 +4,95 @@
...
  change the domain name in the file
  `client/src/environments/environment.prod.ts`. (Specifically, you need to
  change the value of `BASE_URL`.) This value is used to generate the link on
```
Comment (ErikRauer) [inline README.md:98]: Once we merge the two branches, we will want a section linking to SETUP.md

File: README.md (hunk 1 above)
Comment (audreylemeur) [reply README.md:69]: Again? I think it's mentioned in client side

Comment (ErikRauer) [reply README.md:69]: I was thinking that was just a reference to the Auth0 library that we use, not necessarily the fact that we use Auth0

Comment (audreylemeur) [reply README.md:69]: Could you add that?

Comment (ErikRauer) [reply README.md:69]: Yeah I can. Could you move the images to their own directory?

Comment (audreylemeur) [reply README.md:69]: Done

**Pull request merged**